cmd/updater/update.sh

```diff
@@ -664,10 +645,12 @@ if [ ${RESUME_INSTALL} -eq 0 ] && ! $DRYRUN; then
     # Note that the SCRIPTPATH we're passing in should be our binaries directory, which is what we expect to be
     # passed as the last argument (if any)
     echo "Starting the new update script to complete the installation..."
+    echo "copy the update script"
+    cp /home/ubuntu//go-algorand/cmd/updater/update.sh "${UPDATESRCDIR}/bin/${FILENAME}"
```

Comment:
Is ${FILENAME} required for the target of the cp command?

Reply:
I think this is debug code that should be removed. (I believe it was to local copy the update.sh script, but should not be done in final code.)

Reply:
Sorry I couldn't actually use my branch while debugging since I was testing with 2.5.4 code. I must've copy and pasted some wrong changes into the file as I was working on the "not to commit but use to test" and "to-commit" files.

Reply:
Will remove debugging code